src/unity/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -184,13 +165,29 @@ def export_coreml(self, filename, **kwargs):
         filename : string
             The path of the file where we want to save the Core ML model.
 
-        **kwargs : optional Object Detector export_coreml parameters
+        include_non_maximum_suppression : bool
+            Non-maximum suppression is only available in iOS 12+.
+            A boolean parameter to indicate whether the Core ML model should be
+            saved with built-in non-maximum suppression or not. 
+            This parameter is set to True by default.
+
+        iou_threshold : float
+            Threshold value for non-maximum suppression. Non-maximum suppression
+            prevents multiple bounding boxes appearing over a single object. 
+            This threshold, set between 0 and 1, controls how aggressive this 
+            suppression is. A value of 1 means no maximum suppression will 
+            occur, while a value of 0 will maximally suppress neighboring 
+            boxes around a prediction.
+
+        confidence_threshold : float
+            Only return predictions above this level of confidence. The
+            threshold can range from 0 to 1. 
 
         Examples
         --------
         >>> model.export_coreml('one_shot.mlmodel')
         """
-        return self.__proxy__['detector'].export_coreml(filename, **kwargs)
+        return self.__proxy__['detector'].export_coreml(filename, include_non_maximum_suppression, iou_threshold, confidence_threshold)
```

Comment:
It's really not a good idea to pass named/default arguments as positional arguments. If we ever changes the ordering of the named arguments in the object detector `export_coreml(...)`, or we add a new argument not at the end, this will break. Instead please pass the arguments as follows:
```python
include_non_maximum_suppression=include_non_maximum_suppression,
iou_threshold=iou_threshold,
confidence_threshold=confidence_threshold
```